aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -23,6 +32,8 @@ static constexpr int depth_to_max_tensors_scalarlist_of_complex_double[2] = {
     72,
     60};
 
+static constexpr size_t max_kernel_arg_size = 4096;
```

Comment:
just fyi (I don't mean we should update this value at all in this pr)
> CUDA 12.1 increases this parameter limit from 4,096 bytes to 32,764 bytes on all device architectures including [NVIDIA Volta](https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/) and above.

ref: https://developer.nvidia.com/blog/cuda-12-1-supports-large-kernel-parameters/

Reply:
This is awesome!! Thank you for bringing this up. We should adjust this based-on device generation and driver version soon.